lib/log-group.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable no-console */
+
+/**
+ * A simple helper to manage the print of a set of logs
+ */
```

Comment:
This looks public-facing. Can you beef it up? I can't quite infer what I'm supposed to do with this.